Dockerfile

```diff
@@ -28,6 +28,10 @@ RUN apt-get update -qq && \
 # http://unix.stackexchange.com/questions/195975/cannot-force-remove-directory-in-docker-build
 #        && rm -rf /var/lib/apt/lists
 
+# adding custom locales to provide backward support with scrapy cloud 1.0
+RUN echo 'en_US.UTF-8 UTF-8\nde_DE.UTF-8 UTF-8\nnl_NL.UTF-8 UTF-8\nfr_FR.UTF-8 UTF-8' \
+    >> /etc/locale.gen && locale-gen
```

Comment:
I'd suggest changing this a bit - let's move locales list to a file and then copy it to the image before `locale-gen` - I don't like newlines in the string and it'll be easier to maintain this way.

Also I didn't find `/etc/locale.gen` config in `locale-gen` man page - how locale-gen knows it should read list from there?